Vagrantfile

```diff
@@ -16,6 +18,7 @@ Vagrant.configure("2") do |config|
 
   config.vm.define "shell", primary: true do |shell|
     shell.vm.box = "ubuntu/bionic64"
+    shell.vm.disk :disk, size: (ENV['DISK_SIZE'] || "10GB"), primary: true
```

Comment:
@dmartin 
I named the new variable `DISK_SIZE`. However, I only use it for changing the base volume size for the shell server (since web server is untouched at most during a run by users).

This is kind of confusing I realized. Should I rename it to `SHELL_SIZE` or something? What do you think?

Reply:
🤔 Sure, I guess `SHELL_DISK_SIZE` would be clearest. I wasn't worried *too* much about it, as `DISK_SIZE` is already much clearer than the flags that already exist (`J`?)

Reply:
I guess it would be worth cleaning them up. But considering the future roadmap, we'll leave the other flags as is.